src/api/properties/array.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  coerce (val) {
+    return Array.isArray(val) ? val : [val];
+  },
+  default () {
+    return [];
+  },
+  deserialize (val) {
+    return val.split(',');
+  },
+  serialize (val) {
+    return val.join(',');
```

Comment:
This is true. Maybe it's worth being able to specify a `token` or something to `split` or `join` using? It's worth nothing that this should be a simple implementation. If users need complex behaviour, they could throw an abstraction on top of it or write their own property.